user_info.go

```diff
@@ -13,3 +13,20 @@ func (c *WorkwxApp) GetUser(userid string) (*UserInfo, error) {
 	obj := resp.intoUserInfo()
 	return &obj, nil
 }
+
+// GetUserByDept 获取部门成员详情
+func (c *WorkwxApp) GetUserByDeptID(id int64, fetchChild bool) ([]*UserInfo, error) {
```

Comment:
In this library `Get` is used to get a single record, for multiple records we use `List` instead. Also make the parameter name more descriptive:

```suggestion
func (c *WorkwxApp) ListUsersByDeptID(deptID int64, fetchChild bool) ([]*UserInfo, error) {
```

Reply:
I still don't get the reason why the verb `Get` is not changed...